### PR TITLE
Add Samples Index metadata for samples

### PR DESCRIPTION
**Pull request by tjohns**
Add .google/packaging.yaml file to each sample directory, so that the
samples can be picked up by Google's Samples Index system.

Also updated README.md for hello-kotlin sample to add "Kotlin" to the
sample's title.

### CODE REVIEW

File: configuration-apks/.google/packaging.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
+status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
+technologies: [Android]
+categories:   [Instant Apps]
+languages:    [Java]
```
Comment (keyboardsurfer) [inline configuration-apks/.google/packaging.yaml]: Kotlin

File: cookie-api/.google/packaging.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
+status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
+technologies: [Android]
+categories:   [Instant Apps]
+languages:    [Java]
```
Comment (keyboardsurfer) [inline cookie-api/.google/packaging.yaml]: Kotlin

File: install-api/.google/packaging.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
+status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
+technologies: [Android]
+categories:   [Instant Apps]
+languages:    [Java]
```
Comment (keyboardsurfer) [inline install-api/.google/packaging.yaml]: Kotlin

File: storage-api/.google/packaging.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+# Values: {DRAFT | PUBLISHED | INTERNAL | DEPRECATED | SUPERCEDED}
+status:       PUBLISHED
+
+# Optional, put additional explanation here for DEPRECATED or SUPERCEDED.
+# statusNote:
+
+# See http://go/sample-categories
+technologies: [Android]
+categories:   [Instant Apps]
+languages:    [Java]
```
Comment (keyboardsurfer) [inline storage-api/.google/packaging.yaml]: Kotlin

**Review by keyboardsurfer — COMMENTED** [review]
Thanks for the PR.
I added some comments, where the language differs from your proposed change. Please update and this PR is good to go.

File: configuration-apks/.google/packaging.yaml (hunk 1 above)
Comment (tjohns) [reply configuration-apks/.google/packaging.yaml]: Done

File: cookie-api/.google/packaging.yaml (hunk 2 above)
Comment (tjohns) [reply cookie-api/.google/packaging.yaml]: Done

File: install-api/.google/packaging.yaml (hunk 3 above)
Comment (tjohns) [reply install-api/.google/packaging.yaml]: Done

File: storage-api/.google/packaging.yaml (hunk 4 above)
Comment (tjohns) [reply storage-api/.google/packaging.yaml]: Done

**Review by tjohns — COMMENTED** [review]
Changes submitted. PTAL.

**Pull request merged**